### PR TITLE
Rf06 tc027 tc boris medrano 01

### PR DESCRIPTION
**Pull request by borisytu2014**
lista contratos tc
lista contratos

### CODE REVIEW

File: src/main/java/org/umssdiplo/automationv01/core/managepage/Contract/ListContract.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.umssdiplo.automationv01.core.managepage.BasePage;
+import org.umssdiplo.automationv01.core.utils.CommonEvents;
+
+public class ListContract extends BasePage {
+
+    @FindBy(id = "ContractTable")
+    private WebElement paginaContrato;
```
Comment (ctola91) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/Contract/ListContract.java]: Dont use Spanish in code

File: src/main/java/org/umssdiplo/automationv01/core/managepage/Contract/ListContract.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+    public boolean validarListaContrato() {
+        return CommonEvents.isVisible(paginaContrato);
```
Comment (ctola91) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/Contract/ListContract.java]: Change Spanish to English

File: src/main/java/org/umssdiplo/automationv01/core/managepage/Menu/SubMenuPersonalContract.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.umssdiplo.automationv01.core.managepage.BasePage;
+import org.umssdiplo.automationv01.core.managepage.Contract.ListContract;
+import org.umssdiplo.automationv01.core.utils.CommonEvents;
+
+public class SubMenuPersonalContract extends BasePage {
+
+    @FindBy(id = "smContract")
+    private WebElement subMenuContrato;
```
Comment (ctola91) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/Menu/SubMenuPersonalContract.java]: Change Spanish

**Review by ctola91 — CHANGES_REQUESTED** [review]

File: src/main/java/org/umssdiplo/automationv01/core/managepage/Menu/Menu.java [hunk 4]
```diff
@@ -51,4 +51,9 @@ public SubMenuEquipment selectEquipmentMenu() {
         CommonEvents.clickButton(nemuEquipament);
         return new SubMenuEquipment();
     }
+
+    public SubMenuPersonalContract selectPersonalSubMenuContract() {
```
Comment (marbusjim) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/Menu/Menu.java]: change to selectSubMenuPersonalContract

File: src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionSSID.java [hunk 5]
```diff
@@ -238,4 +242,25 @@ public void elResultadoDeBuscarPersonalDeberiaSer(int resultado){
...
+    }
+
+    @And("^seleccionar el submenu 'Contratos' del menu 'Personal'$")
+    public void seleccionarContratos() throws Throwable {
+        listContract = menuPersonalContract.selectSubMenuContract();
+    }
+
+    @Then("^validar que la 'Lista de Contratos' este visible$")
+    public void validarListaDeContratos() throws Throwable {
+        boolean result = listContract.validateContractList();
```
Comment (marbusjim) [inline src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionSSID.java]: You don´t need add a variable, you should use directly in assertion

**Review by marbusjim — APPROVED** [review]
Approve

**Review by danielrpg — APPROVED** [review]
Review Changes

**Pull request merged**